src/index.tsx

```diff
@@ -41,7 +41,7 @@ openSansObserver.load().then(() => {
 const store = configureAppStore();
 const MOUNT_NODE = document.getElementById('root') as HTMLElement;
 
-ReactDOM.render(
+createRoot(MOUNT_NODE!).render(
```

Comment:
update to replace ReactDOM
[ref link](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis)